### ISSUE TITLE
Regarding Hwayoung

### ISSUE THREAD

**Issue by ExFina1** [maphe/maphe.github.io#152]
It is stated that Hwayoung S3 penetrates def. While fiddling with the damage calc I found enemy def still influencing numbers. Is this correct?

**Comment by maphe** [comment]
Can you give more details about what you're testing please? When I try it myself in a basic scenario, the target's defense does not influence S3's damage.

**Comment by duradara** [comment]
> Can you give more details about what you're testing please? When I try it myself in a basic scenario, the target's defense does not influence S3's damage.

I am not op but I also have the same problem. Her S3 penetrates defense but the calculator doest have it.
Here are some images whit different amount of defense, her S3 damage should not change but it does
![Screenshot_20220314-223418](https://user-images.githubusercontent.com/50158352/158288747-0d1c49a6-977e-4fc2-89a5-fdf3bb0a8db3.png)
![Screenshot_20220314-223411](https://user-images.githubusercontent.com/50158352/158288754-83740edc-00de-4236-acfb-054664d240e8.png)

**Comment by maphe** [comment]
@duradara in your case that's because you have Uberius Tooth set up, which only penetrates 70% so there's still 30% of the target's defense going in the artifact damage formula.

**Comment by duradara** [comment]
@maphe ohh I thought the artifact damage was only shown in the top bar. I didn't think it was being added to the table below.

**Comment by Rock30907** [comment]
Dose hwayoung s2 correct ? When I lv up,it goes with lower damage& shield with attack buff, when without attack buff,s2 not even change.
![Screenshot_20220416-111554~2](https://user-images.githubusercontent.com/103802080/163659533-da7b8578-de57-42bf-b7f4-d2000df97fda.png)
![Screenshot_20220416-111546~2](https://user-images.githubusercontent.com/103802080/163659538-3edd92a4-966b-4d90-90b8-53f120f25b6f.png)

**Comment by maphe** [comment]
@Rock30907 yes that's by design, the way the formula works, her passive and the attack buff are additive to each other.

That means if you have 7k attack with S2 at +0 you get more out of attack buff than if you have 7k attack with S2 at +5.

**Comment by Rock30907** [comment]
> @Rock30907 yes that's by design, the way the formula works, her passive and the attack buff are additive to each other.
> 
> That means if you have 7k attack with S2 at +0 you get more out of attack buff than if you have 7k attack with S2 at +5.

Oh I see, it means when you with lower S2 to reach same attack, the attack before buff is higher than max S2 , thanks!
